client/components/contact/ContactIntro.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {
+  Grid,
+} from '@material-ui/core';
+
+const ContactIntro = () => (
+  <Grid container alignItems="center" justify="center" direction="column">
+    <Grid item style={{ width: '45%' }}>
+      <h1 style={{ textAlign: 'center' }}>
+        Don&apos;t See What You Need?
```

Comment:
I think the [first](https://stackoverflow.com/a/40782648) style of including a single quote is more readable